IFC4x3/Properties/h/HasDrainage_30Z3LZrU15p9CBV6oSf4at/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HasDrainage_30Z3LZrU15p9CBV6oSf4at" Name="HasDrainage" UniqueId="c08c3563-d5e0-45cc-930b-7c6c9ca44937" PrimaryDataType="IfcBoolean" />
```

Comment:
rename to HasRailDrainage to allow a more generic HasDrainage if needed anywhere else

Reply:
Makes sense, updated https://github.com/bSI-InfraRoom/IFC-Specification/pull/164/commits/ab85f083c2717d30ef42c6a47dcce352a5b16e89